docs/schema.graphql

```diff
@@ -1,1060 +0,0 @@
-      "transactions": [
-        {
-          "block_hash": "0x47d74ac830a8da437da48d95844a9f60c71eaaeffa9e0547738dd49ffe5417cf",
-          "block_number": 341275,
-          "from_account_id": 27455,
-          "to_account_id": 3014,
-          "type": "POLYJUICE"
-        },
-        {
-          "block_hash": "0xb68eee6a72bfd54a06101bedb264e1026af2228250b82dd7c3f06beb35f5d865",
-          "block_number": 335796,
-          "from_account_id": 172581,
-          "to_account_id": 3014,
-          "type": "POLYJUICE"
-        }
-      ]
-    }
-  }
-  """
-  transactions(input: TransactionInput!): [Transaction]
-
-  """
-  function: get udt by contract address
-  
-  request-example:
-  query {
-    udt(input: {contract_address: "0xbf1f27daea43849b67f839fd101569daaa321e2c"}){
-      id
-      name
-      type
-      supply
-      account{
-        short_address
-      }
-    }
-  }
-  
-  result-example:
-  {
-    "data": {
-      "udt": {
-        "account": {
-          "short_address": "0xbf1f27daea43849b67f839fd101569daaa321e2c"
-        },
-        "id": "1",
-        "name": "Nervos Token",
-        "supply": "693247799.35570027",
-        "type": "BRIDGE"
-      }
-    }
-  }
-  """
-  udt(input: SmartContractInput!): Udt
-
-  """
-  function: get list of udts
-  
-  request-example:
-  query {
-    udts(input: {page: 1, page_size: 2, sort_type: ASC}){
-      id
-      name
-      type
-      supply
-      account{
-        eth_address
-        short_address
-      }
-    }
-  }
-  
-  result-example:
-  {
-    "data": {
-      "udts": [
-        {
-          "account": {
-            "eth_address": null,
-            "short_address": "0xbf1f27daea43849b67f839fd101569daaa321e2c"
-          },
-          "id": "1",
-          "name": "Nervos Token",
-          "supply": "693247799.35570027",
-          "type": "BRIDGE"
-        },
-        {
-          "account": {
-            "eth_address": null,
-            "short_address": "0x21ad25fab1d759da1a419a589c0f36dee5e7fe3d"
-          },
-          "id": "17",
-          "name": null,
-          "supply": "400000002840",
-          "type": "BRIDGE"
-        }
-      ]
-    }
-  }
-  """
-  udts(input: UdtsInput = {page: 1, page_size: 10, sort_type: ASC, type: BRIDGE}): [Udt]
-}
-
-type SmartContract {
-  abi: [Json]
-  account: Account
-  account_id: String
-  compiler_file_format: String
-  compiler_version: String
-  constructor_arguments: String
-  contract_source_code: String
-  deployment_tx_hash: String
-  id: Int
-  name: String
-  other_info: String
-}
-
-input SmartContractInput {
-  contract_address: String!
-}
-
-enum SortType {
-  ASC
-  DESC
-}
-
-type TokenTransfer {
-  amount: Decimal
-  block: Block
-  block_hash: String
-  block_number: Int
-  from_account: Account
-  from_address_hash: String
-  log_index: Int
-  polyjuice: Polyjuice
-  to_account: Account
-  to_address_hash: String
-  token_contract_address_hash: String
-  token_id: Decimal
-  transaction: Block
```

Comment:
Should be `Transaction` here instead of `Block`?

Reply:
this is a bug, i fix this with this pr #395